__tests__/Database.test.js

```diff
@@ -0,0 +1,10 @@
+import 'regenerator-runtime/runtime';
+const itemModel = require('../database/mongooseModel.js');
+
+describe('Database testing', () => {
+  it('should fetch an item from the database', async () => {
+    const random = Math.floor(Math.random() * Math.floor(99));
+    const item = await itemModel.item.findOne({ id: random });
+    expect(typeof item).toEqual('object');
```

Comment:
I would add a test to check item contains a key you're expecting, like 'title' or 'image'.